Source/Core/InputCommon/ControlReference/ControlReference.cpp

```diff
@@ -48,6 +47,17 @@ ParseStatus ControlReference::GetParseStatus() const
   return m_parse_status;
 }
 
+std::string ControlReference::GetExpression() const
+{
+  return m_expression;
+}
+
+void ControlReference::SetExpression(std::string expr)
+{
+  m_expression = expr;
```

Comment:
Can you explain this more? Godbolt's telling me that an explicit `std::move` generates worse code here in both Clang and MSVC.